src/main/kotlin/com/terraformation/backend/customer/db/OrganizationStore.kt

```diff
@@ -148,25 +155,40 @@ class OrganizationStore(
             modifiedBy = currentUser().userId,
             modifiedTime = clock.instant(),
             name = row.name,
+            organizationTypeId = row.organizationTypeId,
+            organizationTypeDetails = row.organizationTypeDetails,
             timeZone = row.timeZone,
+            website = row.website,
         )
 
-    organizationsDao.insert(fullRow)
+    return dslContext.transactionResult { _ ->
+      organizationsDao.insert(fullRow)
 
-    with(ORGANIZATION_USERS) {
-      dslContext
-          .insertInto(ORGANIZATION_USERS)
-          .set(CREATED_BY, currentUser().userId)
-          .set(CREATED_TIME, clock.instant())
-          .set(MODIFIED_BY, currentUser().userId)
-          .set(MODIFIED_TIME, clock.instant())
-          .set(ORGANIZATION_ID, fullRow.id)
-          .set(ROLE_ID, Role.Owner)
-          .set(USER_ID, currentUser().userId)
-          .execute()
-    }
+      with(ORGANIZATION_USERS) {
+        dslContext
+            .insertInto(ORGANIZATION_USERS)
+            .set(CREATED_BY, currentUser().userId)
+            .set(CREATED_TIME, clock.instant())
+            .set(MODIFIED_BY, currentUser().userId)
+            .set(MODIFIED_TIME, clock.instant())
+            .set(ORGANIZATION_ID, fullRow.id)
+            .set(ROLE_ID, Role.Owner)
+            .set(USER_ID, currentUser().userId)
+            .execute()
+      }
 
-    return fullRow.toModel(totalUsers = 1)
+      managedFacilityTypes.forEach {
+        with(ORGANIZATION_MANAGED_FACILITY_TYPES) {
+          dslContext
+              .insertInto(ORGANIZATION_MANAGED_FACILITY_TYPES)
+              .set(ORGANIZATION_ID, fullRow.id)
+              .set(MANAGED_FACILITY_TYPE_ID, it)
```

Comment:
Kotlin style: only use the default `it` lambda function parameter for single-line function bodies. Or in other words, `forEach` should specify a variable name.